app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsRoot.kt

```diff
@@ -75,6 +75,8 @@ fun AgendaDetailsRoot(
     agendaDetailsViewModel: AgendaDetailsViewModel = hiltViewModel()
 ) {
     val viewState by agendaDetailsViewModel.viewState.collectAsStateWithLifecycle()
+    val email by agendaDetailsViewModel.email.collectAsStateWithLifecycle()
+    val isEmailValid by agendaDetailsViewModel.isEmailValid.collectAsStateWithLifecycle()
```

Comment:
Similar question as I had earlier already - why aren't these part of your viewState? The email and whether it's valid definitely has an impact on the UI, right?

Reply:
fixing this !!